src/include/duckdb/main/relation.hpp

```diff
@@ -118,7 +119,8 @@ class Relation : public std::enable_shared_from_this<Relation> {
 	//! Delete from a table, can only be used on a TableRelation
 	DUCKDB_API virtual void Delete(const string &condition = string());
 	//! Create a relation from calling a table in/out function on the input relation
-	DUCKDB_API shared_ptr<Relation> TableFunction(const std::string &fname, const vector<Value> &values);
+	DUCKDB_API shared_ptr<Relation> TableFunction(const std::string &fname, const vector<Value> &values,
```

Comment:
Perhaps leave in the original function that calls TableFunction with an empty set of named_parameters?